python/requirements/development.txt

```diff
@@ -23,7 +23,7 @@ pytest-cov
 pytest-xdist
 python_jsonschema_objects
 PyVCF
-sphinx
+sphinx==3.3.1
```

Comment:
Can we make this `<=3.3.1` so that people who use a system wide env aren't inconvenienced?
```suggestion
sphinx<=3.3.1
```